### DISCUSSION TITLE
Adjust minimum amount of blocks used per claim

### DISCUSSION THREAD

**Discussion by Minecraft-Classic-Server-Archive** [GriefPrevention/GriefPrevention#1812] [Ideas]
Could this be an option? 100 claim blocks minimum is very limiting for smaller builds

**Comment by Minecraft-Classic-Server-Archive** [comment]
Goddammit used a project account

**Comment by bigpresh** [comment]
config setting `GriefPrevention.Claims.MinimumArea`, no?

**Reply by Hedwig7s** [reply]
That sets the minimum area (5x5, 7x7, ect...) 
What I'm referring to is when you create a claim, if the claim will use less than 100 claim blocks, it blocks the claim.

**Reply by bigpresh** [reply]
Because the default value of that setting is 100, no?

Assuming you're getting the error message specified by `Messages.ResizeClaimInsufficientArea` which defaults to:

> This claim would be too small.  Any claim must use at least {0} total claim blocks

(Where the "{0}" is replaced with the value of GriefPrevention.Claims.MinimumArea)

... then it's because `Claims.MinimumArea` is set to a larger value than the area of the claim you're trying to make.

Note that there are two related settings...

* `Claims.MinimumArea`, defaults to 100 blocks, sets the minimum area (x*y) that a claim must cover
* `Claims.MinimumWidth`, defaults to 5 blocks, sets the minimum width either side of the claim could be.

i.e. with the default settings, you can create a 10x10 claim or a 5x20 claim, but can't create a e.g. 2x100 claim.

It really sounds like you just need to adjust `Claims.MinimumArea` to your liking?
